### DISCUSSION TITLE
'Australian Marine Energy Atlas' Service Outage in "National Map"

### DISCUSSION THREAD

**Discussion by MinghanBao** [TerriaJS/terriajs#7266] [Q&A]
I noticed that the data custodian's service is currently experiencing an outage, which has made the 'Australian Marine Energy Atlas' function in the 'National Map' inaccessible. Could you please let me know when this function is expected to be back online? Your assistance in this matter is greatly appreciated.
I’ve attached an image where the error message appears 
<img width="666" alt="Service Outage of National Map" src="https://github.com/user-attachments/assets/e0edd763-10ae-4f1f-b973-da886e195b5a">

**Comment by ljowen** [comment]
Thanks for reporting this @MinghanBao we've got in touch with the group responsible for this service and will update you here on it's status.
